src/main/java/digital/slovensko/autogram/server/AssetsEndpoint.java

```diff
@@ -0,0 +1,65 @@
+package digital.slovensko.autogram.server;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import digital.slovensko.autogram.server.dto.ErrorResponse;
+import digital.slovensko.autogram.server.errors.InvalidUrlParamException;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static java.util.Objects.requireNonNull;
+
+public class AssetsEndpoint implements HttpHandler {
+    private static final List<File> assets = new ArrayList<>();
+
+    static  {
+        var location = AssetsEndpoint.class.getProtectionDomain().getCodeSource().getLocation().getPath();
+        var path = Path.of(location,"digital", "slovensko", "autogram", "server", "assets");
```

Comment:
Toto sa nedá nijako vytiahnuť z aktuálnej classy priamo? Prpadne cez class.getResource() vybrať jeden určite existujúci súbor a pozrieť sa na jeho path?

Reply:
Nedá a vybrať jeden určite existujúci súbor som nechcel, keďže ako tu bolo aj pri swagger súboroch spomínané, dávame do názvu súboru verziu. Mohol by som načítať napríklad index.html a potom tam pridať k tomu `assets/` ale myslím si, že s tým zbytočne getujeme resource a síce by to až tak neovplyvnilo rýchlosť, nie je to takto predsa len lepšie?

Reply:
V navrhovanom stave sa mi nepáči, že tam dávame celú cestu ku classe: "digital", "slovensko", "autogram", "server", "assets". Prakticky je to asi jedno, ale ak by sa classa niekam posúvala alebo by sa z nejakého dôvodu premenoval nejaký package, bolo by to treba aj tu upraviť a zabudlo by sa na to a podobne. Zobrať `index.html` a nahradiť jeho cestu `assets` mi príde ako ok riešenie.

Čo sa týka zbytočného getovania resourcu - "stojí" to prakticky žiadny výkon, keďže sa to deje iba raz pri otvorení docs.

Reply:
To je pravda, na to, že sa to deje len raz som trochu pri odpísaní pozabudol 😅. Upravím 👍